### ISSUE TITLE
[tune] Mute `dill` warnings

### ISSUE THREAD

**Issue by richardliaw** [ray-project/ray#2888]
### Describe the problem
Right now, there are plenty of warning messages from HyperOpt after we set the basicConfig of logger. See below source code logs.

### Source code / logs
```
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
/opt/conda/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
/opt/conda/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
/opt/conda/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
/opt/conda/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
/opt/conda/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
/opt/conda/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
/opt/conda/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
/opt/conda/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
```

Seems to stem here: https://github.com/hyperopt/hyperopt/blob/25eb73600d6113ce928042bf6029a66489d2d55c/hyperopt/main.py#L20-L21

The fix would be most likely to get the HyperOpt logger and set the verbosity to a higher degree.

**Comment by robertnishihara** [comment]
Are we using pytest here? Pytest tends to have pretty low verbosity.

**Comment by richardliaw** [comment]
No, this is an end-to-end example.

**Comment by richardliaw** [comment]
Addressed in #2889
